js/src/views/Signer/components/TransactionMainDetails/transactionMainDetails.js

```diff
@@ -14,52 +14,43 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+import MapsLocalGasStation from 'material-ui/svg-icons/maps/local-gas-station';
```

Comment:
Actually a good point, like it. Also conforms more to our approach of using our components and, as you mentioned, makes sure we have consistency.